documentation/modules/metrics/con-operator-restart-events.adoc

```diff
@@ -0,0 +1,115 @@
+// Module included in the following assemblies:
+//
+// metrics/assembly-metrics.adoc
+
+[id='con-operator-restart-events-{context}']
+
+= Monitoring Strimzi operated Kafka pod restarts
```

Comment:
@PaulRMellor So this is my very first attempt at AsciiDoc, so very keen on your feedback on using it correctly. I've still got content to create (all the TODOs) and will ping you when it's done.